# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/client/TextureClient.h
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f gfx/layers/client/TextureClient.h
--- a/gfx/layers/client/TextureClient.h	Wed Jun 10 21:57:19 2020 +0000
+++ b/gfx/layers/client/TextureClient.h	Wed Jun 10 22:21:02 2020 +0000
@@ -335,7 +335,7 @@
  * lifetime. This means that the lifetime of the underlying shared data
  * matches the lifetime of the TextureClient/Host pair. It also means
  * TextureClient/Host do not implement double buffering, which is the
- * responsibility of the compositable (which would use two Texture pairs).
+ * responsibility of the compositable (which would use pairs of Textures).
  * In order to send several different buffers to the compositor side, use
  * several TextureClients.
  */
@@ -907,6 +907,8 @@
 bool UpdateYCbCrTextureClient(TextureClient* aTexture,
                               const PlanarYCbCrData& aData);
 
+TextureType PreferredCanvasTextureType(const KnowsCompositor&);
+
 }  // namespace layers
 }  // namespace mozilla
 